### PR TITLE
Convert to Vonage gem.

### PR DESCRIPTION
**Pull request by exegete46**
Updated to use the Vonage gem.  Also moved the configuration to a `.env` file, so it is all in one place.  Updated documentation to match.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -37,12 +47,12 @@ def email_insight(phone_info, email)
           :via_options => {
             :address              => "smtp.gmail.com",
             :port                 => 587,
-            :user_name            => 'my@email.com',
-            :password             => MY_PASSWORD,
+            :user_name            => ENV['GMAIL_USERNAME'],
```
Comment (bencgreenberg) [inline app.rb:50]: Can we assign the env variable values to constants and use those constants in the application? It'll reduce the lookup for the value to just the single execution.

Comment (exegete46) [reply app.rb]: Absolutely!  Updated to use Sinatra's settings.  Also refactored out the global `$vonage` variable to a simple caching method.

Comment (bencgreenberg) [reply app.rb]: Awesome, looks great!

**Review by bencgreenberg — APPROVED** [review]
Thanks for this contribution, you really did a great job here!

**Pull request merged**